CHANGELOG

```diff
@@ -52,7 +59,7 @@
 -----------------------------
 
 - new ``--lf`` and ``-ff`` options to run only the last failing tests or
-  "failing tests first" from the last run.  This functionality is provided 
+  "failing tests first" from the last run.  This functionality is provided
```

Comment:
Just as an FYI to why these are here, my editor automatically trims out extra whitespace at the end of lines.